### DISCUSSION TITLE
Does This Library Work with SolidStart?

### DISCUSSION THREAD

**Discussion by p0fi** [ivanhofer/typesafe-i18n#662] [Q&A]
Hi, im wondering if this library works with SolidStart. I tried to setup things and all seems kind of fine. VSCode autocompletes the functions when I use `{LL().HI({name: 'Tom'})}` and stuff. But on the page there is simply no string visible. 

Im wondering if I am doing something wrong when wrapping my application since there is nothing like an `App()` in SolidStart. So I am currently trying to wrap stuff in ` <TypesafeI18n locale='en'>` in my `root.tsx` file. But again: no strings are visible on the page.

**Comment by ivanhofer** [comment]
I never have used `typesafe-i18n` with `SolidStart` before, but in general it should be supported.

You probalby still need to load the locales like here in this example: https://github.com/ivanhofer/typesafe-i18n/blob/main/packages/adapter-solid/example/src/App.tsx#L13-L15

Note: implementing it like in the example will probably nor work if you want to use SSR within your application.
